tests/Behat/DrupalContext.php

```diff
@@ -122,4 +126,21 @@ public function assertAvPortalVideoIframe(string $title): void {
     }
   }
 
+  /**
+   * Presses button with specified id|name|title|alt|value on a given region.
```

Comment:
Presses a button with the specified....

Reply:
The short description should fit in the 80 chars limit and it wouldn't by changing it as requested.